File: runners/python3/sssa-combine.py
# - Python - #

import json

import sys
sys.path.insert(0, './libs/sssa-python')
from SSSA import sssa

fi = open('./tmp/Python3_Combine_Input.json', 'r')
input = fi.read()
args = json.loads(input)
fo = open('./tmp/Python3_Combine_Output.json', 'w')
sss = sssa()
for index,a in enumerate(args):
	args[index] = a.encode('utf-8')

fo.write(json.dumps(sss.combine(args)))
fo.close()

File: runners/ruby/sssa-combine.rb
# - Ruby - #

require 'json'
require_relative '../../libs/sssa-ruby/lib/sssa.rb'

input = File.read('./tmp/Ruby_Combine_Input.json')
args = JSON.parse(input)
o = File.new('./tmp/Ruby_Combine_Output.json', 'w')
o.write SSSA::combine(args).to_json
o.close
